revolv/static/dashboard.scss

```diff
@@ -317,6 +378,138 @@ $sidebar-header-gray: #777;
     // HACK: have to use !important because .rows inside .rows have paddings/margins which have
     // a high specificity, do to their selector being ".row .row".
     padding: 0 !important;
-    margin: 13px 0 0 0 !important;
+    margin: 0 !important;
     height: 300px;
+    background-position: center;
+    background-size: cover;
+}
+
+@media #{$large-up} {
+    .dashboard-project {
+        .dashboard-projct-header {
+            .title-area {
+                .title {
+                    font-size: 22px;
+                    line-height: 22px;
+                }
+                // the project status is inside the title area on large screens,
+                // but outside it next to the location on tablets/phones
+                .project-status-large {
+                    display: inline-block;
+                }
+            }
+        }
+        .controls {
+            text-align: right;
+            form {
+                margin-right: 30px;
+            }
+        }
+    }
+}
+
+@media #{$medium-only} {
+    .dashboard-project {
+        .dashboard-project-header {
+            .project-status-medium {
+                display: inline-block;
+            }
+        }
+        .controls {
+            form {
+                margin-right: 10px;
+            }
+        }
+    }
+}
+
+@mixin mobilfied-dashboard-sidebar {
+    position: fixed;
+    width: 300px;
+    z-index: 200;
+    height: 100%;
+    box-shadow: 0px 0px 22px #222; // to give the "floating" appearance
```

Comment:
Meh, I was thinking that since the shadow gives the sidebar an appearance of floating, it should only show up when it's "detached" aka can be closed